src/System.Runtime.Caching/src/System/Runtime/Caching/UnsafeNativeMethods.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Runtime.ConstrainedExecution;
+using System.Runtime.InteropServices;
+using System.Runtime.Versioning;
+using System.Security;
+using System.Security.Permissions;
+using System.Text;
+
+namespace System.Runtime.Caching
```

Comment:
Content of this file should be moved into \git\corefx\src\Common\src\Interop\Windows and use the same naming convention, ie they go in an Interop class, in the global namespace.

You would then include it in your src\.csproj like
```xml
    <Compile Include="$(CommonPath)\Interop\Windows\kernel32\Interop.CloseHandle.cs">
      <Link>Common\Interop\Windows\kernel32\Interop.GlobalMemoryStatusEx.cs</Link>
    </Compile>
````

Reply:
There is already \Interop\Windows\kernel32\Interop.GlobalMemoryStatusEx.cs with everything I need. Can I reference it in my project?

Reply:
Yes certainly